citi-onboarding/Eventos-CITi
# Feat/create banner

## What I did
1. Create a banner on the site
2. Put two bubbles(blue and green)

## How to test
Open the terminal and go to the folder of this project
In terminal type:
i. git checkout develop
ii. git pull
iii. git checkout feat/create-banner
iv. git pull origin feat/create-why-we-do-it-on-client
v. cd server/
vi. node index.js

Open another terminal at the folder of this projects and type:
i. cd client/
ii. npm run start

In your browser:
1. Open localhost http://localhost:3000
2. See if is similiar to the mockup
3. Open the mockup https://www.figma.com/file/2UJF2r572kDAU4snBxUMwx/citi-eventos?node-id=69%3A0
4. Check if the sizes are responsive for desktop and mobile
